mamba/mamba_env.py

```diff
@@ -75,8 +75,15 @@ def mamba_install(prefix, specs, args, env, *_, **kwargs):
         repo.set_priority(priority, subpriority)
         repos.append(repo)
 
+    
     solver = api.Solver(pool, solver_options)
     solver.add_jobs(specs, api.SOLVER_INSTALL)
+
+    python_version = [irec.version for irec in installed_pkg_recs if irec.name == 'python']
+    if python_version:
+        python_constraint = MatchSpec('python==' + version).conda_build_form()
```

Comment:
yeah this code wasn't tested, right?
version looks undefined here ;)

I have opened a PR with the implementation I had on disk and forgot to commit, sorry about that!

#433

Reply:
Oo I tested it locally and it worked. I guess I forgot to push some commits :/